config/admin-functions/create-machine-cert.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${VX_CONFIG_ROOT:="/vx/config"}"
+: "${VX_METADATA_ROOT:="/vx/code"}"
+
+MACHINE_CERT_PATH="${VX_CONFIG_ROOT}/vx-${VX_MACHINE_TYPE}-cert.pem"
+if [[ "${VX_MACHINE_TYPE}" == "admin" ]]; then
+    MACHINE_CERT_PATH="${VX_CONFIG_ROOT}/vx-${VX_MACHINE_TYPE}-cert-authority-cert.pem"
+fi
```

Comment:
A subtlety here. Whereas I'm calling the VxAdmin cert `vx-admin-cert-authority-cert.pem`, I'm calling the other machines' certs `vx-central-scan-cert.pem`, `vx-mark-cert.pem`, and `vx-scan-cert.pem`. Just trying to make it clear that the VxAdmin cert is capable of issuing other certs (specifically card certs), whereas the other machines' certs are not.

I could of course just call the VxAdmin cert `vx-admin-cert.pem` and have its CA abilities be implied. Or even call all the certs `vx-machine-cert.pem`.

I was just trying to maintain some consistency with how we refer to the VxAdmin cert in vxsuite, where we consistently call it the VxAdmin cert authority cert.